### ISSUE TITLE
Staging Review finding: Question details view doesn't reveal key information from question flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104828]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User can review, edit or confirm their information before form submission.](https://design.va.gov/about/experience-standards/#user-can-review-edit-or-confirm-their-information-before-form-submission)
**VA.gov Experience Standard - category:** [Credibility](https://design.va.gov/about/experience-standards/#category-credibility)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @allison0034 (Design), @briandeconinck (Accessibility)

### Description

When viewing the details of previously submitted questions, the only information available to review is the text of the question. But, there are several flows where users are asked for a subject line for their question, or where users are submitting questions on behalf of others ("business questions"). Without the details of those questions available on the question details page, users may not be sure who or what the questions are about. Users must be able to view the other details they submitted alongside the text of their questions. Without that crucial context, they may not be able to recall key details they have shared, or get a meaningful resolution to their question.

<details><summary>Link, screenshot or steps to recreate</summary>

to be added

</details>

## Recommended action

On the question details page, include information about what users submitted on the form alongside the question, including (if relevent) key information about whom the question concerns, and any other information about the nature of the request.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** [Keep a record of submitted information](https://design.va.gov/patterns/help-users-to/keep-a-record-of-submitted-information)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
